recipes/ccache-toolchain/deactivate.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+export PATH="${_OLD_PATH}"
+unset CCACHE_DIR
```

Comment:
I think it's useful to add a ```ccache -s``` here. Specially on the beginning it will be very helpful to see the statistics to make sure everything is working.